P/PETSc/build_tarballs.jl

```diff
@@ -146,6 +206,13 @@ products = [
 dependencies = [
     Dependency("OpenBLAS32_jll"),
     Dependency("CompilerSupportLibraries_jll"),
+    Dependency("SuperLU_DIST_jll"),
+    Dependency("SuiteSparse_jll"),
+    Dependency("MUMPS_jll"),
+    Dependency("SCALAPACK32_jll"),
+    Dependency("METIS_jll"),
+    Dependency("SCOTCH_jll"),
+    Dependency("PARMETIS_jll"),
```

Comment:
We have lots of incompatible versions of some of these packages (i.e. METIS 4 vs 5, but also others probably), it'd be better to specify compat bounds.

Reply:
done